package.json

```diff
@@ -34,7 +34,7 @@
     "lodash": "^4.17.11",
     "loopback-api-definition": "^3.0.0",
     "loopback-bluemix": "^3.0.0",
-    "loopback-soap": "^1.0.0",
+    "loopback-soap": "github:strongloop/loopback-soap#dremond_fix_soap_gen",
```

Comment:
please revert this change, or change the version accordingly.

Reply:
Yes, once I land the other PR, I will revert this. This PR is dependent on another PR.

Reply:
If this PR depends on another PR then the other PR should be incrementing its version to `1.4.0`. After the other PR merges, this PR should be replacing the `github:...` specifier and incrementing the minimum version of the dependency to match the upstream change (`^1.4.0`).